flask_rest_orm/resources/utils.py

```diff
@@ -22,10 +48,34 @@ def query_from_request(model, request):
     :return: the items for the current page
     """
     builder = model.query
-    for key in request.args:
-        if hasattr(model, key):
-            vals = request.args.getlist(key)  # one or many
-            builder = builder.filter(getattr(model, key).in_(vals))
+
+    def get_operator(attr, key, value):
```

Comment:
This function doesn't need to be an inner function
`attr` should be `model_attr`
`key` should be `operator_key`
`value` should be `filter_value`

Reply:
OK